app/models/reaction.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+# this is model reaction
+class Reaction < ApplicationRecord
+  enum reaction_type: { Like: 0, Sad: 1, Angry: 2, Wow: 3 }
```

Comment:
Ngược rồi em -> key phải là integer 
Về sau giả sử em muốn đổi like thành like2 chẳng hạn thì sẽ không phải update đến database,
Nếu để như này nếu đổi giá trị enum -> phải call update toàn bộ